Content.Server/Administration/Systems/AdminSystem.cs

```diff
@@ -252,8 +256,18 @@ private PlayerInfo GetPlayerInfo(SessionData data, ICommonSession? session)
             overallPlaytime = playTime;
         }
 
-        return new PlayerInfo(name, entityName, identityName, startingRole, antag, roleType, GetNetEntity(session?.AttachedEntity), data.UserId,
-            connected, _roundActivePlayers.Contains(data.UserId), overallPlaytime);
+        return new PlayerInfo(name,
+            entityName,
+            identityName,
+            startingRole,
+            antag,
+            roleType,
+            subtype,
```

Comment:
This does an implicit conversion from `String` to `LocId`, should probably just initalize the subtype as a nullable LocId
`LocId? subtype = null;`